.semaphore/semaphore.yml

```
@@ -0,0 +1,17 @@
+version: v1.0
+name: Initial Pipeline
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+blocks:
+  - name: 'Block #1'
+    task:
+      jobs:
+        - name: specs
+          commands:
+            - bundle exec rake spec
+      prologue:
```

Should we add `env_vars` ?

```suggestion
      env_vars:
        - name: RAILS_ENV
          value: test
      prologue:
```

---

It has no dependancy oh rails so no :kiss